waiter/config-full.edn

```diff
@@ -544,7 +544,13 @@
                                  ;; Unless you've custom-patched your Kubernetes code, then this is probably hard-coded to 5 in the controller code.
                                  ;; https://github.com/kubernetes/kubernetes/blob/207e9d1/staging/src/k8s.io/apiserver/pkg/storage/names/generate.go#L45
                                  :pod-suffix-length 5
-
+                                 :proxy-options {:reverse-proxy-parameters {:cmd "/opt/waiter/envoy/bin/envoy-start"
+                                                                            :cpus "0.1"
+                                                                            :env {}
+                                                                            :flag "REVERSE_PROXY"
+                                                                            :image "twosigma/waiter-envoy"
+                                                                            :mem "256Mi"
+                                                                            :offset 1}}
```

Comment:
I thought of parameters as the equivalent for service parameters with string keys.
As such, `cpus` and `mem` are floating point values.
We could reuse the service description schema to validate this (we should already have a function for it).

`:offset` and `:flag` do not fit as those parameters and should be in a different location.

@DaoWen what do you think?

Reply:
@shamsimam - I feel strongly that this should be formatted exactly as the `:fileserver` entry above for consistency. No `:proxy-options :reverse-proxy` nesting, it should have a `:resources` map, etc.

I agree that `offset` does not belong here. I don't even think that should be a configuration option—we should just know that the offset is 1 based on the fact that we have 1 proxy.

I think the `flag` makes sense here since the `flag` is what controls whether this sidecar is turned on or off.

Reply:
sgtm